aptos-move/framework/aptos-token-objects/sources/aptos_token.move

```diff
@@ -199,14 +199,25 @@ module aptos_token_objects::aptos_token {
         property_types: vector<String>,
         property_values: vector<vector<u8>>,
     ): ConstructorRef acquires AptosCollection {
-        let constructor_ref = token::create_from_account(
-            creator,
-            collection,
-            description,
-            name,
-            option::none(),
-            uri,
-        );
+        let constructor_ref = if (std::features::auids_enabled()) {
```

Comment:
nit: Can you add an import for std::features and just call features::auids_enabled here to keep it consistent?

Reply:
Done.